include/revng/Support/InitRevng.h

```diff
@@ -35,12 +36,17 @@ class InitRevng : public llvm::InitLLVM {
       llvm::cl::HideUnrelatedOptions(CategoriesToHide);
     }
 
+    llvm::BumpPtrAllocator A;
+    llvm::StringSaver Saver(A);
+    llvm::SmallVector<const char *, 0> Arguments(Argv, Argv + Argc);
+    if (auto EnvValue = llvm::sys::Process::GetEnv("REVNG_OPTIONS")) {
+      llvm::cl::TokenizeGNUCommandLine(*EnvValue, Saver, Arguments);
+    }
```

Comment:
Add a comment explaining why we no longer use `ParseCommandLineOptions` for this.

Reply:
Done